includes/Classifai/Providers/OpenAI/Tokenizer.php

```diff
@@ -35,6 +35,32 @@ class Tokenizer {
 	 */
 	public function __construct( $max_tokens ) {
 		$this->max_tokens = $max_tokens;
+
+		/**
+		 * How many characters in one token (roughly)
+		 *
+		 * @since 0.1.0
+		 * @hook classifai_openai_characters_in_token
+		 *
+		 * @param int $characters_in_token How many characters in one token (roughly)
```

Comment:
In order for this information to make it into our documentation correctly, we need curly braces around all the filter information in this file, like `{int}`